taxi/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != 8:
+            raise ValidationError(
+                "License number must consists only of 8 characters!"
+            )
+
+        if not license_number[0:3].isupper():
```

Comment:
that is not working:
`"AB1".isupper()` will return `True`. Add checking about letters